Aquifer/IO.swift

```diff
@@ -12,13 +12,13 @@ import Foundation
 import Swiftz
 
 /// Returns a `Pipe` that reads input from `stdin` line-by-line and terminates on end-of-input.
-public func stdinLn<UO, UI>() -> Proxy<UO, UI, (), String, ()> {
+public func stdinLn() -> Producer<String, ()>.T {
```

Comment:
These need to remain generalized, because the combinator that results in the generalized forms from the non-generalized ones is currently unwriteable AFAICT.